guide/13-managing-arcgis-applications/hub-submodule-intro.ipynb

```diff
@@ -0,0 +1,72 @@
+{
```

Comment:
<p>The <code>hub</code> submodule within <code>arcgis.apps</code> acts as the pythonic interface to <a href="https://www.esri.com/en-us/arcgis/products/arcgis-hub/resources" target="_blank">ArcGIS Hub</a>. It enables automation of several Hub workflows and simplifies the use of the Hub information model by allowing access to Hub objects, such as <code>Initiative</code>, <code>Event</code>, <code>Site</code>, and <code>Page</code>. </p><p>The capabilities supported in the API both match and exceed those available in Hub's UI. It equips users with tools necessary for working with the Sites in their Hub or Enterprise organization. With it, you can create, search for, edit the layout of, and clone sites and pages within the same organization and to different organizations. </p><p>ArcGIS Hub is available with ArcGIS Online in Hub Basic and Hub Premium, depending on your licensing of Hub. Every user of ArcGIS Online has access to Hub Basic, unless upgraded at the Organization level. Users of ArcGIS Enterprise get access to Sites and Pages, similar to Hub Basic users. Once you sign-in to your <code>GIS</code>, you can access the Hub capabilities through <code>gis.hub</code>, which is the entry point to Hub. As an Enterprise Sites user, the entry point is <code>gis.sites</code>. Let's explore this through the architecture diagrams below. </p><p> </p><p>You can learn more about the functionality supported through the classes in the hub submodule through the links below: </p><ul><li><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#hub" target="_blank">Hub</a>: Represents a Hub organization</li><li><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#initiative" target="_blank">Initiative</a>: Represents a Hub Premium Initiative</li><li><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#site" target="_blank">Site</a>: Represents a Hub or Enterprise Site</li><li><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#page" target="_blank">Page</a>: Represents a Page of a Hub or Enterprise Site</li><li><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#event" target="_blank">Event</a>: Represents a Hub Premium Event</li><li>Resource Manager classes for managing the objects above:</li><li class="ql-indent-1"><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#initiativemanager" target="_blank">InitiativeManager</a>: To manage initiatives</li><li class="ql-indent-1"><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#sitemanager" target="_blank">SiteManager</a>: To manage sites</li><li class="ql-indent-1"><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#pagemanager" target="_blank">PageManager</a>: To manage pages</li><li class="ql-indent-1"><a href="https://developers.arcgis.com/python/api-reference/arcgis.apps.hub.html#eventmanager" target="_blank">EventManager</a>: To manage events</li></ul><p>We have three more guides that cover a range of functionality based on Hub type. </p><ul><li>Hub Premium (add link): Working with Initiatives and Events</li><li>Hub Basic (add link): Adding site and page, cloning the site to Hub Premium + Basic + Enterprise, Linking and unlinking pages</li><li>Enterprise Sites (add link): Adding site with custom domain, adding page, editing site layout and theme, editing page layout</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1447/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>